recipes/fswatch/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.17.1":
+    url: "https://github.com/emcrisostomo/fswatch/archive/refs/tags/1.17.1.zip"
+    sha256: "2a2be51807a7138cf50b327a9de29b96892dd357ab3142a5b4e36cb380ee4470"
+  "1.17.1-36":
```

Comment:
Note that this version, as per semver semantics, would come _before_ `1.17.1`, that is, `1.17.1-36 < 1.17.1`, which I don't think is the intention here!

We should clarify if upstream is expecting to release a new version soon with this fix, else we'll need to roll our own schema for this

Reply:
Nice catch, you're absolutely right. Since the latest commit was two years ago and the repository is more or less in hibernation mode, I don't think there will be a tag or release soon. Is there the possibility to refer to a specific [commit](https://github.com/emcrisostomo/fswatch/commit/ba411e0d0fabcd5cbf0881f1380482e2f5ab9f47) without the need to assign it a version?

Reply:
Yes! We can set this version to `1.17.1.cci.YYYYMMDD` for the date of the latest commit, which is our way of indicating that the version comes from a specific commit and is not tagged

Reply:
Thanks, I adapted the `1.17.1-36` version according to the proposed scheme and removed the (faulty) 1.17.1 version.